qoffer-domain/src/main/groovy/life/qbic/portal/portlet/packages/ListPackagesInput.groovy

```diff
@@ -0,0 +1,19 @@
+package life.qbic.portal.portlet.packages
+
+/**
+ * Interface to ask for available service packages.
+ *
+ * This input interface should be implemented by use cases
+ * that want to display that they will list
+ * all available service packages.
+ *
+ * @since 1.0.0
+ */
+interface ListPackagesInput {
+
+  /**
+   * This method triggers the use case <emph>List Packages</emph>.
+   */
+  void listAvailablePackages()
```

Comment:
Nice! I like how you choose the method names for the input and the datasource interfaces 👍 
It's very clear what should happen

Reply:
Thank you a lot for the nice feedback :D